pages/common/tesseract.md

```diff
@@ -2,7 +2,7 @@
 
 > OCR (Optical Character Recognition) engine.
 
-- Recognize text in an image and save it to `output.txt`. The file extension MUST not be mentioned:
+- Recognize text in an image and save it to `output.txt` (the '.txt' extension is added automatically):
```

Comment:
No, I'm going by @adius' comments in #1267:

> ...it MUST be `output` and not `output.txt`...

In any case, this change doesn't affect the meaning -- it's ust a rephrasing for clarity and to use less threatening language. I'll see if I can test it, though.